mechanics/RELEASE-VERSIONING-PRINCIPLES.md

```diff
@@ -46,7 +46,7 @@ Upstream reference: https://kubernetes.io/releases/patch-releases
 
 ### Knative community support window principle:
 
-We (the community) support the most recent 4 versions of Knative. The term
+We (the community) support the most recent 2 versions of Knative. The term
```

Comment:
I think we agreed to change this to 6 months, rather than a number of releases, so that we don't need to do future gymnastics here.

Reply:
That's boil down to the same outcome. 2 releases * 3 months apart.

Maybe better to keep saying about number of releases, because that's allow to calculate EOL easily, using Cron expression as well.

EOL for n release: Look for n+2 release date and add one week.

Reply:
If we mean the "add a week" thing to be real, we should spell it out here.

I'm okay with that, though the fact that we don't support skip-level upgrades makes that extra week less meaningful.

Reply:
I added such sentence on line 15 in `mechanics/RELEASE-SCHEDULE.md`.